vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-operators.xml

```diff
@@ -263,10 +263,10 @@
 
     <!-- op:is-same-node($parameter1 as node(), $parameter2 as node()) as xs:boolean -->
     <operator name="op:is-same-node">
-        <param name="parameter1" type="node()?"/>
-        <param name="parameter2" type="node()?"/>
+        <param name="parameter1" type="node()"/>
+        <param name="parameter2" type="node()"/>
         <return type="xs:boolean"/>
-    </operator>
+     </operator>
```

Comment:
Added an extra space. Make sure your eclipse is set up to format XML documents correctly.